tests/wp-tests/Util/CheckCheckboxTest.php

```diff
@@ -0,0 +1,28 @@
+<?php namespace EmailLog\Util;
+
+/**
+ * Test `checked_array` functions inside helper.
+ */
+class CheckCheckboxTest extends \WP_UnitTestCase {
```

Comment:
Rename the class and file to `CheckedArrayTest`

Reply:
Fixed